lib/Dashboard.dart

```diff
@@ -8,8 +10,125 @@ class Dashboard extends StatefulWidget {
 }
 
 class _DashboardState extends State<Dashboard> {
+  int _selectedIndex = 0;
+  var _currentIndex = 0;
+  void _onItemTapped(int index) {
+    setState(() {
+      _selectedIndex = index;
+    });
+  }
+/*
+  @override
+  void initState() {
+    super.initState();
+    BackButtonInterceptor.add(myInterceptor);
+  }
+
+  @override
+  void dispose() {
+    BackButtonInterceptor.remove(myInterceptor);
+    super.dispose();
+  }
+
+  bool myInterceptor(bool stopDefaultButtonEvent, RouteInfo info) {
+    print("BACK BUTTON!"); // Do some stuff.
+    return true;
+  }
+
+ */
+
   @override
   Widget build(BuildContext context) {
-    return Container();
+    const TextStyle optionStyle =
+        // TextStyle(fontSize: 30, fontWeight: FontWeight.bold);
+        TextStyle(fontFamily: 'outfit', fontSize: 15);
+    const List<Widget> _widgetOptions = <Widget>[
+      Text(
+        'Index 0: Home',
+        style: optionStyle,
+      ),
```

Comment:
You could have a function that takes in String. So, you don't have to declare style 4 times.